### ISSUE TITLE
tesstrain.sh doesn't support vertical languages

### ISSUE THREAD

**Issue by davidb1** [tesseract-ocr/tesseract#2989]
### Environment

* **Tesseract Version**: 
```
tesseract 5.0.0-alpha-685-g3a3c4
 leptonica-1.75.3
  libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.5.2) : libpng 1.6.34 : libtiff 4.0.9 : zlib 1.2.11 : libwebp 0.6.1 : libopenjp2 2.3.0
```
* **Platform**: Ubuntu 18.04

### Current Behavior:
When passing a _vert language to tesstrain.sh in --lang

> tesstrain.sh --lang lang_vert

it throws an error:

> ERROR: Error: lang_vert is not a valid language code

as per this line https://github.com/tesseract-ocr/tesseract/blob/d8d2f6f48a8ddaf0b668eb1abf18fd6d08470041/src/training/language-specific.sh#L1170

### Expected Behavior:
Throw a more specific error: `ERROR: Error: vertical languages aren't supported`
or add a config to generate data for vertical languages

### Suggested Fix:
Add a case for _vert languages in https://github.com/tesseract-ocr/tesseract/blob/d8d2f6f48a8ddaf0b668eb1abf18fd6d08470041/src/training/language-specific.sh

**Comment by Shreeshrii** [comment]
Vertical languages seem to be supported indirectly based on font names.

Please see:

https://github.com/tesseract-ocr/tesseract/blob/d8d2f6f48a8ddaf0b668eb1abf18fd6d08470041/src/training/language-specific.sh#L862-L869

and

https://github.com/tesseract-ocr/tesseract/blob/d8d2f6f48a8ddaf0b668eb1abf18fd6d08470041/src/training/tesstrain_utils.sh#L274-L281

Try adding your font to the vertical fonts list as well as the language fonts list and try.

**Comment by davidb1** [comment]
@Shreeshrii so the **_vert** languages are made just by training on vertical fonts only or are there additional steps?

**Comment by Shreeshrii** [comment]
I have not trained any CJK languages or any other scripts requiring vertical fonts. I just pointed out what I found by searching on vert in the training script.

I sy you give it a try. You can add more vertical fonts to the list if needed.

**Comment by Shreeshrii** [comment]
You can try contacting @zodiac3539 for pointers, see https://github.com/zodiac3539/jpn_vert

**Comment by davidb1** [comment]
> You can try contacting @zodiac3539 for pointers, see https://github.com/zodiac3539/jpn_vert

I actually did try a couple months back but he doesn't wanna part with his secrets :)

**Comment by kamui-7** [comment]
Having the same issue here

**Comment by Shreeshrii** [comment]
Please see comment by Ray at https://github.com/tesseract-ocr/tesseract/issues/707#issuecomment-278376168

so, it's possible that the current code is using layout analysis for vertical text rather than a separate language.

**Comment by Shreeshrii** [comment]
Fixed via PR #3223

See https://github.com/tesseract-ocr/tesseract/issues/3001 for discussion
